python/screen_recorder_sdk/screen_recorder.py

```diff
@@ -57,8 +55,12 @@ def get_instance (cls):
         return cls.__instance
 
     def __init__ (self):
-
-        self.lib = ctypes.cdll.LoadLibrary (pkg_resources.resource_filename (__name__, os.path.join ('lib', 'ScreenRecorder.dll')))
+        try:
+            dll_path = importlib.resources.files(__package__).joinpath('lib/ScreenRecorder.dll')
+        except Exception:
+            import importlib.resources as resources
```

Comment:
it supports only windows, but still, is it better to join paths for lib and actuall dll properly?